tests/tests.bats

```diff
@@ -142,15 +142,15 @@ test_install_one_version()
 }
 
 # First real release made this way
-@test "tarball_multi 2.2.2" {
-    local ver=v2.2.2
+@test "tarball_multi 2.2.3" {
+    local ver=v2.2.3
```

Comment:
You cannot test a tarball before it has been released. Keep testing 2.2.2 here, don't switch to 2.2.3 that should not exist yet. This 2.2.2 test should fail because you're adding some files to the sof-v2.2 directory. In this test code, delete the files you're adding now and the test should pass. See similar test hack example in commit https://github.com/thesofproject/sof-bin/commit/601be1094e17bb9485fd83a7e838de65ba8342c9